packages/flutter/lib/src/material/page.dart

```diff
@@ -44,11 +44,14 @@ class MaterialPageRoute<T> extends PageRoute<T> {
     @required this.builder,
     RouteSettings settings,
     this.maintainState = true,
+    Duration transitionDuration = const Duration(milliseconds: 300),
```

Comment:
Are the changes in this file supposed to be here? They seem unrelated...

Reply:
Oops, these are unrelated and are part of something else I'm working on. I'll remove these.